src/main/java/executor/command/CommandHelp.java

```diff
@@ -1,32 +1,51 @@
 package executor.command;
 
 import executor.task.TaskList;
+import interpreter.Parser;
 import ui.Ui;
 import ui.Wallet;
 
+import java.security.spec.ECField;
+import java.util.ArrayList;
+import java.util.Collections;
+
 public class CommandHelp extends Command {
 
     /**
      * Constructor to provide the user with the details about the commands available.
      */
     public CommandHelp(String userInput) {
         this.userInput = userInput;
-        this.description = "Provides the user with all the available commands and descriptions.";
+        this.description = "Provides the user with all the available commands and descriptions.\n"
+                + "FORMAT : help \n";
         this.commandType = CommandType.HELP;
     }
 
     @Override
     public void execute(TaskList taskList) {
-        for (String s : CommandType.getNames()) {
-            if (!s.equals("ERROR") && !s.equals("TASK") && !s.equals("BLANK")) {
-                CommandType commandType = CommandType.valueOf(s);
-                Command c = Executor.createCommand(commandType, "null");
-                String commandDesc = c.getDescription();
-
-                System.out.println(s.toUpperCase() + " - " + commandDesc);
-            }
+        String checkerForExtraInput;
+        checkerForExtraInput = Parser.parseForPrimaryInput(CommandType.HELP, userInput);
+
+        if (!checkerForExtraInput.isEmpty()) {
+            Ui.dukeSays("Extra variable is added. Please follow -> FORMAT : help");
+            return;
         }
-        Ui.printSeparator();
+
+          ArrayList<String> help = new ArrayList<String>();
+          for (String s : CommandType.getNames()) {
+              if (!s.equals("ERROR") && !s.equals("TASK") && !s.equals("BLANK")) {
+                  CommandType commandType = CommandType.valueOf(s);
+                  Command c = Executor.createCommand(commandType, "null");
+                  String commandDesc = c.getDescription();
+                  String temp = s.toUpperCase() + " - " + commandDesc;
```

Comment:
could add String temp = s.toUpperCase() + " - " + commandDesc + "\n"; if you really wanted a newline between each Command.

Reply:
Managed to input that :)